dev-docs/bidders/nasmediaAdmixer.md

```diff
@@ -24,16 +24,13 @@ prebid_1_0_supported : true
 
 ### Note:
 
-The Nasmedia Admixer Bidder Adapter requires setup and approval from the Nasmedia Admixer team.  
-Please reach out to <prebid@nasmedia.co.kr> for more information. 
+The Nasmedia Admixer Bidder Adapter requires setup and approval from the Nasmedia Admixer team.
+Please reach out to <prebid@nasmedia.co.kr> for more information.
 
 ### bid params
 
 {: .table .table-bordered .table-striped }
-
-| Name       | Scope    | Description                                    | Example         |
-| :--------- | :------- | :--------------------------------------------- | :-------------- |
-| `ax_key`   | required | Publisher Key provided by Nasmedia Admixer     |`'ajj7jba3'`     |
-
+| Name     | Scope    | Description                                | Example      | Type     |
+|----------|----------|--------------------------------------------|--------------|----------|
+| `ax_key` | required | Publisher Key provided by Nasmedia Admixer | `'ajj7jba3'` | `string` |
 ---
```

Comment:
Can you take a look at this one? The table isn't displaying correctly when I preview it on the site. Maybe it needed that extra line break? Or the underline below is causing a problem?